documentation/docs/19-accessibility.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Accessibility
+---
+
+SvelteKit strives to provide an accessible platform for your app by default. Svelte's [compile-time accessibility checks](https://svelte.dev/docs#accessibility-warnings) will also apply to any SvelteKit application you build. 
+
+Here's how SvelteKit's built-in accessibility features work and what you need to do to help these features to work as well as possible.
+
+We recognize that accessibility can be hard to get right. If you want to suggest improvements to how SvelteKit handles accessibility, please [open a GitHub issue](https://github.com/sveltejs/kit/issues).
+
```

Comment:
I'm not really sure we need this sentence. It could apply to any section as we want people to open issues about issues they encounter anywhere.
```suggestion
```

Reply:
[comment] I thought this bit was thoughtful; it's signaling to the accessibility community that their feedback is welcome, which is a welcome exception in tech (by large, over the last 30 years). The sentence may not be a technical necessity but it might be a community necessity. Just a perspective to consider. 👍

Reply:
I can go either way on this, but Melanie captures why I included it in the first place. It emphasizes that we want to be receptive to a11y suggestions and improve as needed.

Reply:
Agree with keeping it